mod/05_standard/set/right/account.rb

```diff
@@ -33,7 +33,7 @@ def authenticate_by_token val
   end
 
   view :reset_password_url do |args|
-    wagn_url "/update/#{card.cardname.url_key}?reset_token=#{card.token_card.refresh(true).content}"
+    wagn_url "update/#{card.cardname.url_key}?reset_token=#{card.token_card.refresh(true).content}"
```

Comment:
eg, if site root is mydomain.com/wagn, then this needs to be mydomain.com/wagn/update, not mydomain.com/update